fotoapparat/src/main/java/io/fotoapparat/result/Photo.kt

```diff
@@ -62,13 +62,10 @@ data class Photo(
     }
 
     companion object {
-
-        private val EMPTY by lazy { Photo(encodedImage = ByteArray(0), rotationDegrees = 0) }
```

Comment:
There is not need to keep this here since it creates extra fields and methods since it's only used in tests.

Reply:
👍